src/components/shared/Button/Close.css

```diff
@@ -24,6 +24,11 @@
   margin-top: 0;
 }
 
+.close-btn:hover {
+  display: block;
+  background: var(--theme-selection-background);
```

Comment:
This is already present on line 37. You probably want to have a keyboard focus styling instead.